### DISCUSSION TITLE
Fine-grained PAT vs. classical?

### DISCUSSION THREAD

**Discussion by MattKindzerske** [machulav/ec2-github-runner#131] [General]
The 'main' branch of this repo has a readme describing the input `github-token` required as "GitHub Personal Access Token with the repo scope assigned."  Verifying that this works with the 'classical' PATs.  But (so far) I'm unable to get a fine-grained PAT to work, if this project would worked with a fine-grained PAT and what scoping would need to be assigned.

**Comment by machulav** [comment]
The action was not tested with "Fine-grained personal access tokens" as they're still in beta testing. 
Please use classic tokens instead.

You can create a new issue for that in the repository if you need it. Based on the other users' feedback, we will handle it if it is high enough.
